views/event-detail.html

```diff
@@ -0,0 +1,34 @@
+{% extends "layout.html" %}
+
+{% block mainContent %}
+  <div>
+    <p>What: {{event.title}}</p>
+    <p>When: {{event.date}}</p>
+    <p>Where: {{event.location}}</p>
+    <p>With whom: {{event.attending}}</p>
+    <p>Why: {{event.image}}</p>
+  </div>
+  
+  <div>
+      <p>Do you want to partake in our revelry? Then RSVP!</p>
+      <form> 
+          <method="POST"></method>
+          <div>
+              <label for="email">E-mail:</label>
+              <input type="email" id="email" />
+          </div>
+          
+          <div class = "button">
```

Comment:
No spaces before and after `=`.  Curious as to why this `div` is needed at all.  The submission should be accomplished via an `input` element of `type='submit'`.  See the form @francescoannetta created in #24 for an example.